modules/oadp-1-5-0-release-notes.adoc

```diff
@@ -0,0 +1,223 @@
+
+With this update, you must use the new  `spec.configuration.nodeAgent` parameter in a Data Protection Application (DPA) for configuring the `nodeAgent` `daemonset`.
+Velero no longer uses the `node-agent-config` ConfigMap resource for configuring the `nodeAgent` `daemonset`.
+
+link:https://issues.redhat.com/browse/OADP-5042[OADP-5042]
+
+.Configuring DPA with with the backup repository configuration configMap is now possible
+	
+With Velero 1.15 and later, it is now possible to configure the total size of a cache per repository. This prevents pods from being removed due to running out of ephemeral storage. It also adds the following new parameter to a DPA of `NodeAgentConfig`:
+
+* `cacheLimitMB`: It sets the local data cache size limit in megabytes.
+* `fullMaintenanceInterval`: The default value is 24 hours. Controls the removal rate of deleted Velero backups from the Kopia repository using the following override options:
+** `normalGC: 24 hours`
+** `fastGC: 12 hours`
+** `eagerGC: 6 hours`
+
+link:https://issues.redhat.com/browse/OADP-5900[OADP-5900]
+
+.Enhancing the node-agent security
+
+With this update, the following changes are added:
+
+* A new `configuration` option is now added to the `velero` field in the DPA specification.
+* The default value for the `disableFsBackup` parameter is `false` or `non-existing`, which is the same as false. With this, the following options are added to the `SecurityContext`:
+** `Privileged: true`
+** `AllowPrivilegeEscalation: true`
+* With the `disableFsBackup`parameter set to `true`, removes the following mounts from the node-agent:
+** `host-pods`
+** `host-plugins`
+* Modifies that the node-agent is always run as a non-root user.
+* Changes the root file system to read only.
+* Updates the following mount points with the write access:
+** `/home/velero`
+** `tmp/credentials`
+* Uses the `SeccompProfileTypeRuntimeDefault` option for the `SeccompProfile` parameter.
+
+link:https://issues.redhat.com/browse/OADP-5031[OADP-5031]
+
+.Adds DPA support for parallel item backup
+
+By default, only one thread processes an item block. Velero 1.16 supports a parallel item backup, where multiple items within a backup can be processed in parallel. 
+You can use the optional Velero server parameter `--item-block-worker-count`to run additional worker threads to process items in parallel. To enable this in OADP, set the `dpa.Spec.Configuration.Velero.ItemBlockWorkerCount` parameter to an integer value greater than zero.
+
+link:https://issues.redhat.com/browse/OADP-5635[OADP-5635]
+
+.OADP logs are now available in the JSON format
+
+With the of release {oadp-short} 1.5.0, the logs are now available in the JSON format. It helps to have pre-parsed data in their Elastic logs management system.
+
+link:https://issues.redhat.com/browse/OADP-3391[OADP-3391]
+
+.The `oc get dpa` command now displays Status
+
+With this release, the `oc get dpa` command now displays Status to improve user experience.
```

Comment:
```suggestion
With this release, the `oc get dpa` command now displays status to improve user experience.
```

Same comment: The status name seems to be missing here. Consider mentioning some details how it improves user experience.